docs/en/operations/system_tables.md

```diff
@@ -482,6 +482,76 @@ When the table is deleted manually, it will be automatically created on the fly.
+
+Columns:
+
+- `event_date` (Date) — Event date.
+- `event_time` (DateTime) — Event time.
+- `query_start_time` (DateTime) — Start time of query execution.
+- `query_duration_ms` (UInt64) — Duration of query execution.
+- `read_rows` (UInt64) — Number of read rows.
+- `read_bytes` (UInt64) — Number of read bytes.
+- `written_rows` (UInt64) — For `INSERT` queries, the number of written rows. For other queries, the column value is 0.
+- `written_bytes` (UInt64) — For `INSERT` queries, the number of written bytes. For other queries, the column value is 0.
+- `memory_usage` (Int64) — Memory consumption by the whole query.
+- `peak_memory_usage` (Int64) — Maximum memory consumption by the thread.
+- `thread_name` (String) — Name of the thread function.
+- `thread_number` (UInt32) — Internal thread ID.
+- `os_thread_id` (Int32) — OS thread ID.
+- `master_thread_number` (UInt32) — Internal ID of initial thread.
+- `master_os_thread_id` (Int32) — OS initial thread ID.
+- `query` (String) — Query string.
+- `is_initial_query` (UInt8) — Query type. Possible values:
+    - 1 — Query was initiated by the client.
+    - 0 — Query was initiated by another query for distributed query execution.
+- `user` (String) — Name of the user who initiated the current query.
+- `query_id` (String) — ID of the query.
+- `address` (FixedString(16)) — IP address the query was initiated from.
+- `port` (UInt16) — The server port that was used to receive the query.
+- `initial_user` (String) —  Name of the user who ran the parent query (for distributed query execution).
+- `initial_query_id` (String) — ID of the parent query.
+- `initial_address` (FixedString(16)) — IP address that the parent query was launched from.
+- `initial_port` (UInt16) — The server port that was used to receive the parent query from the client.
+- `interface` (UInt8) — Interface that the query was initiated from. Possible values:
+    - 1 — TCP.
+    - 2 — HTTP.
+- `os_user` (String) — User's OS.
+- `client_hostname` (String) — Server name that the [clickhouse-client](../interfaces/cli.md) is connected to.
+- `client_name` (String) — The [clickhouse-client](../interfaces/cli.md) name.
+- `client_revision` (UInt32) — Revision of the [clickhouse-client](../interfaces/cli.md).
+- `client_version_major` (UInt32) — Major version of the [clickhouse-client](../interfaces/cli.md).
+- `client_version_minor` (UInt32) — Minor version of the [clickhouse-client](../interfaces/cli.md).
+- `client_version_patch` (UInt32) — Patch component of the [clickhouse-client](../interfaces/cli.md) version.
+- `http_method` (UInt8) — HTTP method that initiated the query. Possible values:
+    - 0 — The query was launched from the TCP interface.
+    - 1 — `GET` method was used.
+    - 2 — `POST` method was used.
+- `http_user_agent` (String) — The `UserAgent` header passed in the HTTP request.
+- `quota_key` (String) — The quota key specified in the [quotas](quotas.md) setting.
```

Comment:
Slightly incorrect. This is the user specified arbitrary string that is used as the key for quota.